spec/interactors/create_user_session_spec.rb

```diff
@@ -0,0 +1,34 @@
+require "rails_helper"
+
+RSpec.describe CreateUserSession do
+  subject { described_class }
+
+  let(:user) { create(:unconfirmed_user) }
```

Comment:
It looks like you are not using this guy so feel free to finish him :smile: :facepunch: